conan/tools/cmake/cmakedeps.py

```diff
@@ -98,44 +96,31 @@
 set({name}_COMPILE_OPTIONS{build_type_suffix}_LIST "{deps.cxxflags_list}" "{deps.cflags_list}")
 set({name}_COMPILE_OPTIONS_C{build_type_suffix} "{deps.cflags_list}")
 set({name}_COMPILE_OPTIONS_CXX{build_type_suffix} "{deps.cxxflags_list}")
-set({name}_LIBRARIES_TARGETS{build_type_suffix} "") # Will be filled later, if CMake 3
-set({name}_LIBRARIES{build_type_suffix} "") # Will be filled later
-set({name}_LIBS{build_type_suffix} "") # Same as {name}_LIBRARIES
+set({name}_LIB_DIRS{build_type_suffix} {deps.lib_paths})
+set({name}_LIBS{build_type_suffix} {deps.libs})
 set({name}_SYSTEM_LIBS{build_type_suffix} {deps.system_libs})
 set({name}_FRAMEWORK_DIRS{build_type_suffix} {deps.framework_paths})
 set({name}_FRAMEWORKS{build_type_suffix} {deps.frameworks})
-set({name}_FRAMEWORKS_FOUND{build_type_suffix} "") # Will be filled later
 set({name}_BUILD_MODULES_PATHS{build_type_suffix} {deps.build_modules_paths})
+"""
 
-conan_find_apple_frameworks({name}_FRAMEWORKS_FOUND{build_type_suffix} "${{{name}_FRAMEWORKS{build_type_suffix}}}" "${{{name}_FRAMEWORK_DIRS{build_type_suffix}}}")
 
-mark_as_advanced({name}_INCLUDE_DIRS{build_type_suffix}
```

Comment:
Is this needed?

Reply:
yes, `mark_as_advanced` is just not to pollute CMakeGUI